woocommerce/woocommerce
# Fix: Product collection inspector control showing empty section

### Changes proposed in this Pull Request:





Closes # .

This PR fixes the issue where the inspector control was showing an empty PanelBody element as shown in screenshot below:

<img width="882" alt="image" src="https://github.com/user-attachments/assets/804a8d36-0268-4378-a974-ab6fc03f5528">

This PR also add placeholder for "Hand-picked Products" control to improve user experience.



### How to test the changes in this Pull Request:



**1. Verify that the inspector control is not showing an empty section**
1. Create a new post and add a Product Collection block.
2. Select any collection, except for "Hand-picked Products", for example "Featured Products"
3. Verify that the inspector control is not showing an empty section.

| Before | After |
|--------|--------|
| <img width="882" alt="Before - Empty section in inspector control" src="https://github.com/user-attachments/assets/804a8d36-0268-4378-a974-ab6fc03f5528"> | <img width="283" alt="After - Fixed inspector control" src="https://github.com/user-attachments/assets/fe7d1966-dcd7-4285-a32c-babbef5736bf"> |


**2. Verify that the "Hand-picked Products" control is showing placeholder text**
1. Create a new post and add a Product Collection block.
2. Select "Hand-picked Products" collection.
3. Verify that following placeholder text is showing as shown below: "Search for products to display…" 
<img width="408" alt="image" src="https://github.com/user-attachments/assets/834c0701-2c3e-45e1-a21f-e7cb67dc76c1">




### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 
Product Collection: Fix inspector control showing empty section
</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
